pkgs/top-level/python-packages.nix

```diff
@@ -12381,14 +12381,14 @@ in {
   pygit2 = callPackage ../development/python-modules/pygit2 { };
 
   Babel = buildPythonPackage (rec {
```

Comment:
It would be nice if you could move this expression to `pkgs/development/python-modules/Babel/default.nix`.